test-common/test-common-runtime/src/main/java/org/apache/kafka/common/test/ClusterInstance.java

```diff
@@ -444,6 +444,16 @@ default List<Integer> boundPorts() {
                 .map(KafkaBroker::socketServer)
                 .map(s -> s.boundPort(clientListener()))
                 .toList();
+    }
 
+    default void restartDeadBrokers() throws InterruptedException {
+        if (brokers().isEmpty()) {
+            throw new IllegalArgumentException("Must supply at least one server config.");
+        }
+        brokers().entrySet().forEach(entry -> {
+            if (!entry.getValue().isShutdown()) {
```

Comment:
it seems the method is used to restart "dead" brokers, so the condition should be `if (entry.getValue().isShutdown())`, right?

Reply:
Thanks for the feedback. I've addressed issues mentioned above.